### PR TITLE
Polkadot XCM Body constants

### PR DESCRIPTION
**Pull request by muharem**
Constants for OpenGov

### CODE REVIEW

File: runtime/polkadot/constants/src/lib.rs [hunk 1]
```diff
@@ -98,6 +98,18 @@ pub mod fee {
 	}
 }
 
+/// XCM protocol related constants.
+pub mod xcm {
+	/// Pluralistic bodies existing within the consensus.
+	pub mod body {
+		// Preallocated for the Root body.
+		#[allow(dead_code)]
+		const ROOT_INDEX: u32 = 0;
```
Comment (bkchr) [inline runtime/polkadot/constants/src/lib.rs:107]: Make it also public directly?

**Review by bkchr — APPROVED** [review]

**Comment by muharem** [comment]
bot rebase

**Review by joepetrowski — APPROVED** [review]

**Comment by muharem** [comment]
bot merge

**Comment by muharem** [comment]
bot rebase

**Comment by muharem** [comment]
bot merge

**Pull request merged**